research-software-directory/RSD-as-a-service
# Software and project overview by person based on ORCID

# People overview

Closes #879

Changes proposed in this pull request:
* If the contributor have ORCID we enable link to "people" page 
* The visitors can view all software and projects of "ORCID" person by clicking on the name
* The info about role, affiliation and email of all entries is grouped and listed on the page

How to test:
* `make start` to build app
* navigate to software page, select one software item and go to contributors section. Click on the contributor name. It should take you to a page showing software collection of this subject (based on ORCID). 
* find a person without ORCID (or create one). The link should not be placed on this person name (because we can only filter by ORCID)
* navigate to a project and perform the similar operation with team members
* NOTE! test data is not best suited for this. It is better to restore backup of real data and test it locally.

## Link to specific person contributions
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/e5d3ba3e-c394-40fc-bcf2-37493e0360ce)

## Contributor page
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/c1cfd2cd-9f12-47db-b886-c89236d717c6)

PR Checklist:
